src/components/fx/hover.js

```diff
@@ -339,8 +338,15 @@ function _hover(gd, evt, subplot, noHoverEvent) {
         // the rest of this function from running and failing
         if(['carpet', 'contourcarpet'].indexOf(trace._module.name) !== -1) continue;
 
-        subplotId = helpers.getSubplot(trace);
-        subploti = subplots.indexOf(subplotId);
+        if(trace.type === 'splom') {
+            // splom traces do not generate overlay subplots,
+            // it is safe to assume here splom traces correspond to the 0th subplot
+            subploti = 0;
+            subplotId = subplots[subploti];
```

Comment:
OK for now, but we should remove this restriction at some point - it would be weird to overlay a splom on something else, but you never know what weird use cases users will come up with.

Reply:
If ok, I'd leave out splom trace on overlaid axes from this PR.